static/app/components/events/autofix/autofixRootCause.tsx

```diff
@@ -165,15 +166,37 @@ export function replaceHeadersWithBold(markdown: string) {
 
 function RootCauseDescription({cause}: {cause: AutofixRootCauseData}) {
   return (
-    <Fragment>
-      <CauseDescription
-        dangerouslySetInnerHTML={{
-          __html: marked(replaceHeadersWithBold(cause.description)),
-        }}
-      />
+    <CauseDescription
+      dangerouslySetInnerHTML={{
+        __html: marked(replaceHeadersWithBold(cause.description)),
+      }}
+    />
+  );
+}
+
+function RootCauseContext({
+  cause,
+  repos,
+}: {
+  cause: AutofixRootCauseData;
+  repos: AutofixRepository[];
+}) {
+  const unitTestFileExtension = cause.unit_test?.file_path
+    ? getFileExtension(cause.unit_test.file_path)
+    : undefined;
+  const unitTestLanguage = unitTestFileExtension
+    ? getPrismLanguage(unitTestFileExtension)
+    : undefined;
+
+  return (
+    <RootCauseContextContainer>
       {cause.reproduction && (
         <Fragment>
-          <ExpandableInsightContext title={'How to reproduce'}>
+          <ExpandableInsightContext
+            icon={<IconLaptop size="sm" color="subText" />}
```

Comment:
Why a laptop? I don't have a suggestion but maybe there's a better icon for this

Reply:
There's a long line of icons that somewhat make sense, maybe the info icon instead?
![CleanShot 2024-10-01 at 14 53 18@2x](https://github.com/user-attachments/assets/75eebae3-37f7-4c14-9dc6-17bdba0095f4)
![CleanShot 2024-10-01 at 14 53 05@2x](https://github.com/user-attachments/assets/1d3a6a52-7b53-4a9f-a5b9-005897e998cd)
![CleanShot 2024-10-01 at 14 52 54@2x](https://github.com/user-attachments/assets/08a7b62e-5f8c-4363-87d7-86de763d5d4c)
![CleanShot 2024-10-01 at 14 52 46@2x](https://github.com/user-attachments/assets/00b67501-d42c-4c93-82a4-943ea231ad94)
![CleanShot 2024-10-01 at 14 52 43@2x](https://github.com/user-attachments/assets/17a98f5d-8a33-48d5-8c13-63490a010005)
![CleanShot 2024-10-01 at 14 52 31@2x](https://github.com/user-attachments/assets/30df7f50-d06d-4e04-9674-2527457a1144)
![CleanShot 2024-10-01 at 14 52 14@2x](https://github.com/user-attachments/assets/71184679-250a-4ded-932a-9fec46ac3b6b)
![CleanShot 2024-10-01 at 14 52 08@2x](https://github.com/user-attachments/assets/eb31a12c-44b1-49f4-b44a-e9f257a8241a)

Reply:
🤷 Maybe ask Vu, or just use the laptop as a placeholder for now until we can get his input

Reply:
We'll just do info for now and ill follow up for input